src/Core/src/Handlers/IViewHandler.Android.cs

```diff
@@ -5,8 +5,6 @@ namespace Microsoft.Maui
 {
 	public interface IAndroidViewHandler : IViewHandler
 	{
-		void SetContext(Context context);
```

Comment:
This is not needed because the Android `Context` is part of the `IMauiContext` which is already attached to the handler. @PureWeen @rmarinho this is also why I put the `IMauiContext` on the handlers because we were attaching the native `Context` here. If we move the MAUI context, then this may or may not need to also be considered.